skaffold.yaml

```diff
@@ -7,13 +7,12 @@ build:
     envTemplate:
       template: "{{.VERSION}}"
   artifacts:
-  - image: valory/open-autonomy-open-aea
-    context: autonomy/data/Dockerfiles/agent/
+  - image: valory/open-autonomy
+    context: deployments/Dockerfiles/autonomy
     docker:
       dockerfile: Dockerfile
       buildArgs:
         AEA_VERSION: "1.16.0"
-        AEA_AGENT: "{{.AEA_AGENT}}"
```

Comment:
@DavidMinarsch this fixes the issue caused by missing agent id while running `skaffold build`, we don't need the agent id to build the image anymore.